examples/rpl_udp/helper.c

```diff
@@ -148,7 +150,9 @@ void rpl_udp_ignore(int argc, char **argv)
     tcmd.transceivers = TRANSCEIVER_CC1100;
     tcmd.data = &a;
 
-    printf("sending to transceiver (%" PRIkernel_pid "): %u\n", transceiver_pid,
-           (*(uint8_t *)tcmd.data));
+    a = atoi(argv[1]);
+    printf("sending to transceiver (%u): %u\n", transceiver_pid, (*(uint8_t *)tcmd.data));
```

Comment:
(Moreover: the a is set in https://github.com/RIOT-OS/RIOT/pull/2602/files#diff-774299209354f6ffeb33ebb74a48b5c9R144 already).